plugins/package-managers/node/src/funTest/assets/projects/synthetic/pnpm-project-with-lockfile-expected-output.yml

```diff
@@ -1,6 +1,6 @@
 ---
 project:
-  id: "PNPM::pnpm-test:1.0.0"
+  id: "PNPM::pnpm-package-with-lockfile:1.0.0"
```

Comment:
Commit message: Typo in "nname".

As the title is the same as for the first commit, it would be nice to somehow mention Yarn2 / PNPM as part of the each title. Maybe simply use them as the scope instead of "node"?